peers.go

```diff
@@ -15,68 +15,101 @@ limitations under the License.
 */
 
 // peers.go defines how processes find and communicate with their peers.
+// Each running Universe instance is a peer of each other, and it has
+// authority over a set of keys, or "stars", within each galaxy (address
```

Comment:
Let's drop the `stars` reference. Having one random reference like this is likely to cause confusion more than chuckles.
It might work if this line read like:
```suggestion
// authority over a set of keys (stars?) within each galaxy (address
```

This way it's clear that we don't intend to make everyone use that terminology, and is really mostly extending the metaphor.